bodhi/server/views/metrics.py

```diff
@@ -23,8 +23,10 @@ def compute_ticks_and_data(db, releases, update_types):
     """ Return the data and ticks to make the stats graph. """
     data, ticks = [], []
 
-    for i, release in enumerate(sorted(releases, cmp=lambda x, y:
-            cmp(int(x.version_int), int(y.version_int)))):
+    releases = sorted(releases, cmp=lambda x, y:
+            cmp(int(x.version_int), int(y.version_int)))
```

Comment:
At first I didn't see the difference, but now I see that there's another call to ```enumerate(releases)``` a few lines down, and that line didn't have the sorting that we see here.

I think this change makes sense, thanks for the contribution!